datafusion/core/tests/physical_optimizer/aggregate_statistics.rs

```diff
@@ -0,0 +1,306 @@
+
+use datafusion_common::config::ConfigOptions;
```

Comment:
Moved into its own file for consistency. If people like this pattern I can do it for the other tests too